ingest/build-configs/ncbi/bin/ncbi-virus-url

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+"""
+Generate URL to download all virus sequences and their curated metadata for a
+specified NCBI Taxon ID from GenBank via NCBI Virus.
+
+The URL this program builds is based on the URL for SARS-CoV-2 constructed with
+
+    https://github.com/nextstrain/ncov-ingest/blob/2a5f255329ee5bdf0cabc8b8827a700c92becbe4/bin/genbank-url
+
+and observing the network activity at
+
+    https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide
+"""
+from urllib.parse import urlencode
+from typing import List, Optional
+import argparse
+
+def parse_args():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--ncbi-taxon-id", required=True,
+        help="NCBI Taxon ID. Visit NCBI virus at " +
+             "https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/find-data/virus " +
+             "to search for supported taxon IDs.")
+    parser.add_argument("--filters", required=False, nargs="*",
+        help="Filter criteria to add as `fq` param values. " +
+             "Apply filters via the NCBI Virus UI and observe the network " +
+             "activity to find the desired filter string.")
+    parser.add_argument("--fields", required=False, nargs="*",
+        help="Metadata fields to add as `fl` param values. " +
+             "Expected to be formatted as <output_column_name>:<ncbi_virus_field_name>. " +
+             "See docs/ncbi-virus-all-fields-example.json for the available NCBI Virus fields.")
+    return parser.parse_args()
+
+def build_query_url(ncbi_taxon_id: str,
+                    filters: Optional[List[str]]=None,
+                    fields: Optional[List[str]]=None):
```

Comment:
```suggestion
def build_query_url(ncbi_taxon_id: str,
                    filters: Optional[List[str]]=None,
                    fields: Optional[List[str]]=None) -> None:
```

Reply:
Included in https://github.com/nextstrain/avian-flu/pull/40/commits/eb5e068ef2d4260b01c7ac49fa805b539fb05e83